src/Help/UpdateHelp/Program.cs

```diff
@@ -152,7 +152,8 @@ static void TestModeWarning()
             string[] targetFiles = Directory.GetFiles(targetPath, "*.htm", SearchOption.AllDirectories);
             foreach (string file in targetFiles)
             {
-                if (file.EndsWith("index1.htm"))
+                // auxiliary html files with titles or headings that can be ignored
+                if (file.EndsWith("index1.htm") || file.EndsWith("csh-redirect.htm"))
```

Comment:
What do you think about this? I would require the addition of `using System.Linq;` at the top of the file.
``` suggestion
            // auxiliary html files with titles or headings that can be ignored
            string[] ignoreFiles = { "index1.htm", "csh-redirect.htm" };
            foreach (string file in targetFiles)
            {
                if (ignoreFiles.Contains(Path.GetFileName(file)))
```